trajectory_utils/src/trajectory_utils/conversions/conversions.cpp

```diff
@@ -170,8 +170,8 @@ void time_to_speed_constjerk(const std::vector<double>& downtrack, const std::ve
       cur_speed = prev_speed;
     }
     
-    if(delta_d == 0){
-      cur_speed =0;
+    if(delta_d == 0.0001){
```

Comment:
This is not what i meant, you need a range such as abs(delta_d) < 0.0001. Direct comparison with 0.0001 is no better than ==0